buddyboss/buddyboss-platform
# Fixed Edit Album button size issue

### All Submissions:


Fixes #513 

### How to test the changes in this Pull Request:

1. Go to any Album Page
2. Edit Album 
3. Check album rename buttons

### Proof Screenshots or Video
https://prnt.sc/u3is1y

### Other information:
